### PR TITLE
Return Active Actors count in gRPC metadata API

### PR DESCRIPTION
**Pull request by berndverst**
Signed-off-by: Bernd Verst <4535280+berndverst@users.noreply.github.com>

# Description

Even though the actors API is http only, the Metadata API can provide information about hosted actors via gRPC. This change includes the missing information about active actors.

Note that some SDKs such as Python generally query APIs via gRPC and hence this implementation is necessary.

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [x] Unit tests passing
* [x] End-to-end tests passing

### CODE REVIEW

**Comment by berndverst** [comment]
cc @halspang @artursouza @tmacam

File: pkg/grpc/api_test.go [hunk 1]
```diff
@@ -1,3 +1,6 @@
+//go:build unit
+// +build unit
```
Comment (halspang) [inline pkg/grpc/api_test.go:2]: I think I may have been mistaken when I said you needed this here. I think you just needed the `-tags=unit` in the command to run the test.

I don't think there's a problem with having it, but we should see if we need it.

**Review by halspang — COMMENTED** [review]
Code looks fine to me, just a question about whether we need the thing I told you to add offline :)

File: pkg/grpc/api_test.go (hunk 1 above)
Comment (berndverst) [reply pkg/grpc/api_test.go]: confirmed not needed

**Review by halspang — APPROVED** [review]

**Comment by tmacam** [comment]
LGTM

**Comment by berndverst** [comment]
This always fails on a different Kind test. The tests are flaky but that has nothing to do with this PR :)
And Fossa is flakey as everyone knows. Their API seems to be having trouble again.

**Pull request merged**